build/ci.go

```diff
@@ -608,7 +608,7 @@ func doDocker(cmdline []string) {
 			}
 			if mismatch {
 				// Build numbers mismatching, retry in a short time to
-				// avoid concurrent failes in both publisher images. If
+				// avoid concurrent files in both publisher images. If
```

Comment:
Is this supposed to be `fails` or `files`? Just want to be sure.

Reply:
I think fails is correct here